Regular-Question-Answers/isSubsequence.cpp

```diff
@@ -0,0 +1,33 @@
+// Author: Bilgenur Çelik
+// Question Link: https://leetcode.com/problems/is-subsequence/
+
+class Solution {
+public:
+    bool isSubsequence(string s, string t) {

```

Comment:
Please describe what string s and t are, and what the question wants to do with them briefly